recipes/f77_zmq/LICENSE

```diff
@@ -0,0 +1,495 @@
+GNU LESSER GENERAL PUBLIC LICENSE
```

Comment:
The LICENSE file is already included in the release tarball. We prefer that you use that one instead of including it here in the recipe.